main/2022/CG-05-10.md

```diff
@@ -41,3 +41,271 @@ Installation is required, see the calendar invite.
 ## Meeting Notes
 
 To be added after the meeting.
+### Attendees
+
+* Yurii Rashkovskii
+* Francis McCabe
+* Yury Delendik
+* Zalim Bashrov
+* Ezzat Chamudi
+* Eric Prud’Hommeaux
+* Ashley Nelson
+* Jeff Charles
+* Dan Gohman
+* Francis McCabe
+* Sam Clegg
+* Yuri Iozzelli
+* Chris Fallin
+* Paolo Severini
+* Slava Kuzmich
+* Rick Battagline
+* Ilya Rezvov
+* Jacob Abraham
+* Michal Kawalec
+* Charles Vaughn
+* Keith Miller
+* Ryan Hunt
+* Asumu Takikawa
+* Ben Titzer
+* Luke Wagner
+* Mingqui Sun
+* Igor Iakovlev
+* Johnnie Birch
+* Heejin Ahn
+* Andreas Rossberg
+* Sebastien Deleuze
+* Peter Heune
+* rrw
+* Sean Jensen-Grey
+* Deepti Gandluri
+* Kevin Moore
+* Andrew Brown
+* Paul Schoenfelder
+* Charles Vaughn
+* Sergey Rubanov
+* Alex Chricton
+* Jakob Kummerow
+* George Kulakowski
+
+
+### Discussion
+
+#### Update on [Memory64](https://github.com/WebAssembly/memory64/) (Sam Clegg) [10 min]
+
```

Comment:
Here are the slides: https://docs.google.com/presentation/d/1yBBmBRobBWcl3_WvtOacPxnpCqmZk1fbK8K9RNgbpz8/edit?usp=sharing

Reply:
Thanks!